lte/gateway/python/magma/pipelined/app/inout.py

```diff
@@ -87,6 +88,7 @@ def __init__(self, *args, **kwargs):
         # following fields are only used in Non Nat config
         self._dhcp_gw_info = None
         self._gw_mac_monitor = None
+        self.redis_client = get_default_client()
```

Comment:
default client does not use redis_config, so there could be some inconsistency in redis clients used.
I think we need to change get_default_client() to use redis config, so that every component on AGW uses same config, rather than just mobilityD.

Reply:
I am fine you breaking this change in different patches. You can get rid of all get_default_client() from mobilityD, its CLI and inout APP in pipelineD for now.